docs/DEV.md

```diff
@@ -10,15 +10,22 @@
   - [Creating extension builds for distribution](#creating-extension-builds-for-distribution)
   - [Development mode](#development-mode)
     - [Firefox](#firefox)
```

Comment:
should Firefox and Chrome be removed here, leaving only three flavours?

Reply:
Yes! Perfect typo catch :)